conbench/api/compare.py

```diff
@@ -160,16 +179,13 @@ def __init__(
         )
 
     @property
-    def unit(self) -> str:
-        if self.baseline and self.baseline.unit:
-            return self.baseline.unit
-        if self.contender and self.contender.unit:
-            return self.contender.unit
-        return "unknown"
-
-    @property
-    def less_is_better(self) -> bool:
-        return _less_is_better(self.unit)
+    def less_is_better(self) -> Optional[bool]:
+        """
+        This is only defined when this comparison has a unit.
```

Comment:
I also added this specification here because I think it makes sense.

Reply:
Thank you!